Structural/Decorator/Webservice.php

```diff
@@ -21,7 +21,7 @@ public function __construct($data)
     }
 
     /**
-     * @return string
+     * @return mixed
```

Comment:
Yes, it should. But the interface declaration says mixed, too. I'll close here and fix it to be consistently `string`.